tests/unit/copydb.sh

```diff
@@ -45,7 +43,20 @@ do
     t=`basename $f .sql`
     r=/tmp/results/${t}.out
     e=./expected/${t}.out
-    psql -d "${PGCOPYDB_TARGET_PGURI}" ${pgopts} --file ./sql/$t.sql &> $r
+    psql -d "${PGCOPYDB_TARGET_PGURI}" ${pgopts} --file $f &> $r
```

Comment:
Please revert that change, I don't think we need to change anything on that line (looks like noise now).